example/docker-compose-prod/docker-compose.yml

```diff
@@ -0,0 +1,130 @@
+                            "answerDelayBetweenAttempt": 750,
+                            "avatarAPI": "https://robohash.org/[user_id]?set=set4&bgset=bg2&size=320x240",
+                            "callDelay": 250,
+                            "delayBeforeClosingCall": 1000
+                        },
+                        "meet": {
+                            "serverURL": "jitsi.YOURDOMAIN.COM",
+                            "roomDefaultName": "lemverse-test"
+                        },
+                        "character": {
+                            "walkSpeed": 180,
+                            "runSpeed": 720
+                        },
+                        "characterNames": ["Basic", "Ghost"],
+                        "skins": {
+                            "guest": "Ghost",
+                            "default": "Basic"
+                        },
+                        "templateLevelId": "lvl_iLOVEaLOTlemverse"
+                    },
+                    "defaultLevelId": "lvl_iLOVEaLOTlemverse",
+                    "peer": {
+                        "path": "/peer",
+                        "client": {
+                            "url": "peer.YOURDOMAIN.COM",
+                            "port": 443,
+                            "credentialDuration": 86400,
+                            "secret": "",
+                            "config": { 
+                                "iceServers": [{ 
+                                "urls": "stun:stun.l.google.com:19302"
+                                }],
+                                "iceTransportPolicy" : "all",
+                                "sdpSemantics": "unified-plan"
+                            }
+                        },
+                        "server": {
+                            "start": false,
+                            "port": 443,
+                            "key": "peerjs",
+                            "alive_timeout": 60000,
+                            "expire_timeout": 5000,
+                            "allow_discovery": false
+                        }
+                    }
+                }
+    peer:
+        image: peerjs/peerjs-server
+        container_name: peer
+        restart: unless-stopped
+        networks:
+            - net_lemverse
+        command: [ "--port", "9000", "--path", "/peer" ]]
+    caddy:
+        image: caddy/caddy:alpine
+        restart: unless-stopped
+        container_name: caddy
+        environment:
+            FRONTEND_HOST: app.YOURDOMAIN.COM
+            JITSI_HOST: jitsi.YOURDOMAIN.COM
+            PEER_HOST: peer.YOURDOMAIN.COM
+            JITSI_SERVICE: jitsi
+            FRONTEND_SERVICE: lemverse
+            PEER_SERVICE: peer
+        ports:
+            - "80:80"
+            - "443:443"
+        networks:
+            - net_lemverse
+        volumes:
+            - ./Caddyfile:/etc/caddy/Caddyfile:ro
+            - caddy:/data/caddy:z
+
+volumes:
```

Comment:
Volumes (and globally containers or another things in a project) should be prefixed by an `lemverse-` to prevent conflict with another project. I think is a good thing.
It is better to understand what is `lemverse-db` that `db` when you perform the command :  `docker volume ls` 😊

Reply:
<img width="681" alt="Screenshot 2021-11-04 at 01 12 10" src="https://user-images.githubusercontent.com/4084527/140236113-901a0c84-4223-4fa4-ac01-4bf1c3e945b1.png">
Using `docker-compose` volumes are auto prefixed with the folder na,e as you can see in my screenshot.
agree for image otherwise

Reply:
Yeah! Knowing it for container but not for volume. It makes sense. Thank you 👍